app_flutter/lib/state/flutter_build.dart

```diff
@@ -68,6 +76,9 @@ class FlutterBuildState extends ChangeNotifier {
     /// [Timer.periodic] does not necessarily run at the start of the timer.
     _fetchBuildStatusUpdate();
 
+    _fetchFlutterBranches()
+        .then((List<String> branchResponse) => _branches = branchResponse);
```

Comment:
Is it possible to use different `refreshRate` for these two functions? `branch` shall not be changed frequently.

Reply:
This doesn't use a timer or refresh for `_fetchFlutterBranches`. `_fetchBuildStatusUpdate` has a `Timer.periodic` does whereas the current implementation of `_fetchFlutterBranches` just requests the branches once at start of `FlutterBuildState`.